code/modules/mob/living/ventcrawling.dm

```diff
@@ -31,10 +31,14 @@
 			to_chat(src, span_warning("You can't vent crawl while buckled!"))
 		return
 	if(iscarbon(src) && required_nudity)
```

Comment:
``required_nudity`` exists man

Reply:
that's fully naked like God intended, this is a separate balance change to ventcrawling in general